test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class MiniGMPTestConan(ConanFile):
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    generators = 'cmake'
+
+    def imports(self):
+        self.copy('*.dll', src='bin', dst='bin')
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
```

Comment:
As always, I find it safer if the test binary is installed before being executed

Reply:
In principle I agree, I will do once we finalised a solution for `RPATH` issues. Right now I don't think it is time well spent to fiddle around with something that will change anyway soon.